src/jaegertracing/samplers/Config.cpp

```diff
@@ -15,12 +15,39 @@
  */
 
 #include "jaegertracing/samplers/Config.h"
+#include "jaegertracing/utils/EnvVariable.h"
 
 namespace jaegertracing {
 namespace samplers {
 
 constexpr double Config::kDefaultSamplingProbability;
 constexpr const char* Config::kDefaultSamplingServerURL;
 
+constexpr const char* Config::kJAEGER_SAMPLER_TYPE_ENV_PROP;
+constexpr const char* Config::kJAEGER_SAMPLER_PARAM_ENV_PROP;
+constexpr const char* Config::kJAEGER_SAMPLER_MANAGER_HOST_PORT_ENV_PROP;
+
+void Config::fromEnv()
+{
+    const auto samplerType = utils::EnvVariable::getStringVariable(kJAEGER_SAMPLER_TYPE_ENV_PROP);
+    if (!samplerType.empty()) {
+        _type = samplerType;
+    }
+
+    const auto param = utils::EnvVariable::getStringVariable(kJAEGER_SAMPLER_PARAM_ENV_PROP);
+    if (!param.empty()) {
+        std::istringstream iss(param);
+        int paramVal = 0;
+        if (iss >> paramVal) {
+            _param = paramVal;
+        }
+    }
+
+    const auto samplerUrl = utils::EnvVariable::getStringVariable(kJAEGER_SAMPLER_MANAGER_HOST_PORT_ENV_PROP);
```

Comment:
fyi https://github.com/jaegertracing/jaeger-client-go/issues/282#issuecomment-537415291

we may want to avoid introducing a name that will be changed/deprecated soon

Reply:
what do you prefer? we can skip it or eventually use the name.

Reply:
maybe exclude it from this PR?

Reply:
done